### ISSUE TITLE
Fix the mapping of data into tables when exporting ideas to CSV

### ISSUE THREAD

**Issue by Artmir** [linnovate/openideal#153]
The Openideal feature to export content (ideas) to csv: admin/openideal/activity/content/idea exports all data in the same cell. I believe some cell remapping needs to be done. The comma doesn't seem to seperate content. 
![Capture](https://user-images.githubusercontent.com/13940618/110624493-f0d55880-81a6-11eb-8f8e-027ad71e67b8.JPG)

**Comment by Artmir** [comment]
Changing the delimiter from the default comma (,) -> smi column (;), fixes the spacing issue. However when opened with excel the columns need to be expanded by hand. Furthermore, there is no formatting kept. Only plain text data.

**Comment by Artmir** [comment]
I suggest to add the following feature to the next openideal version:
Currently the options to export data is limited to CSV, Json, XML etc. However, in order to format the cells correctly I propose to add the ability to export data in latest (xlsx) fomat as well.  I did this by installing the 'Serialisation (excel)' module, which integrates with the serialisation. Tested it and exports views data perfectly to an excel sheet.
